frontend/openchat-sw/src/sw/sw.ts

```diff
@@ -14,12 +14,18 @@ registerRoute(
         return [
             /assets\/.*png|jpg|svg|gif|mp4/,
             /main-.*[css|js]$/,
-            /ArchitecturePage-.*js$/,
-            /HashLinkTarget-.*js$/,
-            /Headline-.*js$/,
-            /RoadmapPage-.*js$/,
-            /SMSUpgrade-.*js$/,
-            /WhitepaperPage-.*js$/,
+            /vendor-.*[css|js]$/,
+            /ArchitecturePage-.*[css|js]$/,
```

Comment:
The new CSS files are output into the assets directory.
We should either bring them up to root or handle them being in assets here.